owa/app/js/components/manage_autogeneration_option/helpers.js

```diff
@@ -0,0 +1,80 @@
+import axios from "axios";
```

Comment:
why do you need this when you already have apiHelper.js ?

Reply:
I was already using this before @adamsdenniskariuki informed me of`apiHelper.js`. And I find this straightforward and easy to use

Reply:
it's a reppitition since it does the same thing

Reply:
how hard is apiCall(data, requestType, url) compared to this down here?